### PR TITLE
Add progress chart to content page in activities table

### PR DESCRIPTION
**Pull request by charvp**
image

image

### CODE REVIEW

File: config/locales/views/activities/en.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ en:
       submissions: Submissions for this exercise
       edit: Edit exercise
     activity:
-      count_tooltip: number of students that solved this exercise correctly
+      count_tooltip: Number of students that solved this exercise correctly
+      read_tooltip: Number of students that read this reading activity
```
Comment (bmesuere) [inline config/locales/views/activities/en.yml:10]:
```suggestion
      read_tooltip: Number of students that read this
```

File: config/locales/views/activities/nl.yml [hunk 2]
```diff
@@ -6,7 +6,9 @@ nl:
       edit: Oefening bewerken
       submissions: Oplossingen voor deze oefening
     activity:
-      count_tooltip: aantal studenten die deze oefening correct hebben
+      count_tooltip: Aantal studenten die deze oefening correct hebben
+      read_tooltip: Aantal studenten die deze leesactiviteit gelezen hebben
```
Comment (bmesuere) [inline config/locales/views/activities/nl.yml:10]:
```suggestion
      read_tooltip: Aantal studenten die dit gelezen hebben
```

File: config/locales/views/activities/nl.yml [hunk 3]
```diff
@@ -41,6 +43,9 @@ nl:
       progress_chart_info_correct:
         one: "Één gebruiker loste ze correct op."
         other: "%{count} gebruikers losten ze correct op."
+      progress_chart_info_read:
+        one: "Één gebruiker las deze leesactiviteit."
```
Comment (bmesuere) [inline config/locales/views/activities/nl.yml:47]:
```suggestion
        one: "Één gebruiker heeft dit gelezen."
```

File: config/locales/views/activities/nl.yml [hunk 4]
```diff
@@ -41,6 +43,9 @@ nl:
...
+        other: "%{count} gebruikers lazen deze leesactiviteit."
```
Comment (bmesuere) [inline config/locales/views/activities/nl.yml:48]:
```suggestion
        other: "%{count} gebruikers hebben dit gelezen."
```

**Comment by charvp** [comment]
> How do these suggestions sound?

That's what I did at first, but I didn't really like the difference between the strings for exercise and for content. But I don't have a strong opinion.

**Pull request merged**